lib/Search/Query/AQueryParameter.php

```diff
@@ -37,7 +37,7 @@ class AQueryParameter {
 	
 	public function getValue() {
 		if (is_string($this->value) && mb_strlen($this->value) > 1) {
-			$param = ($this->value[0] === '"' && $this->value[mb_strlen($this->value) - 1] === '"') ? mb_substr($this->value, 1, -1): $this->value;
+			$param = ($this->value[0] === '"' && $this->value[strlen($this->value) - 1] === '"') ? mb_substr($this->value, 1, -1): $this->value;
```

Comment:
why do you use the non-mb function just once? should it also be changed on the line before and the same line after?

Reply:
Yep that was intentional to get the byte at the offset position, but thinking about it it might be a bit safer to rewrite with using a multibyte substring for those. Pushed a fix and test cases.